.pre-commit-config.yaml

```diff
@@ -12,14 +12,14 @@ repos:
     -   id: rust-style
         name: Rust style
         description: Run cargo fmt on files included in the commit. rustfmt should be installed before-hand.
-        entry: cargo +nightly-2021-11-30 fmt --all -- --check
+        entry: cargo fmt --all -- --check
```

Comment:
This change was introduced in the [PR](https://github.com/openebs/mayastor/pull/1052) to make use of `spdk-rs` crate. Wont switching it back cause issues in those set of commits in develop?

Reply:
No, this was previously reverted in cfe04ce1, it mainly affects those (like me) who don't use the `rustup` toolchain. The `+xyz` bit actually applies to `rustup` and so doesn't work if you don't have it installed. Without that, it simply runs the current default toolchain.